spec/collator.html

```diff
@@ -187,7 +177,7 @@
       </p>
 
       <ul>
-        <li>The first element of [[sortLocaleData]][locale].co and [[searchLocaleData]][locale].co must be null for all locale values.</li>
+        <li>The first element of [[sortLocaleData]][locale].co and [[searchLocaleData]][locale].co must be *null* for all locale values.</li>
```

Comment:
The word null must be formatted bold, not as code (IIRC, "*" make code formatting?)